### PR TITLE
More dependabot fixes

### PR DESCRIPTION
**Pull request by GnaneshKunal**
Fixes

- tough-cookie
- semver
- protobufjs

### CODE REVIEW

**Review by vlad-dargel — CHANGES_REQUESTED** [review]
@GnaneshKunal can you return back testcafe version (discard tests/e2e/package.json and tests/e2e/yarn.lock)?
there is an issue in testCafe 3.1.0+ https://github.com/DevExpress/testcafe/issues/7886 which fails our 23 e2e tests

**Review by vlad-dargel — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -82,7 +82,9 @@
     "**/node-sass": "^8.0.0",
     "**/trim": "0.0.3",
     "word-wrap": "1.2.4",
-    "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5"
+    "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5",
+    "**/semver": "^7.5.2",
+    "rawproto/protobufjs": "^7.2.4"
```
Comment (romansergeenkosofteq) [inline package.json:87]: Is this version compatible with "rawproto": "^0.7.6"?

Comment (vlad-dargel) [reply package.json:87]: @GnaneshKunal

Comment (GnaneshKunal) [reply package.json:87]: Looking into this. Basic RI UI test did pass. Running tests/same code from the library

Comment (GnaneshKunal) [reply package.json:87]: Ran rawproto tests. Worked fine for protobuf `^7.2.4`

**Pull request merged**